### PR TITLE
[221/222] Make add role/subject buttons secondary

### PR DESCRIPTION
**Pull request by elliotcm**
### Before
Screenshot 2023-03-16 at 09 02 18

Screenshot 2023-03-16 at 09 07 18

### After
Screenshot 2023-03-16 at 09 03 42

Screenshot 2023-03-16 at 09 08 17

### CODE REVIEW

**Comment by elliotcm** [comment]
[Add role should be a secondary button on the work history page](https://trello.com/c/jb6XCnhJ/221-add-role-should-be-a-secondary-button-on-the-work-history-page)

**Comment by elliotcm** [comment]
[Use secondary button styling for ‘add another subject’](https://trello.com/c/fhYyyZWS/222-use-secondary-button-styling-for-add-another-subject)

**Comment by bencmitchell** [comment]
Thanks @elliotcm , these changes all look good to go 👍

File: app/views/jobseekers/profiles/employments/review.html.slim [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
     h1.govuk-heading-l = t(".title")
 
-    = govuk_button_link_to(t(".add_role_button_text"), new_jobseekers_profile_work_history_path)
+    = govuk_button_link_to(t(".add_role_button_text"), new_jobseekers_profile_work_history_path, class: "govuk-button--secondary")
```
Comment (josephhull676) [inline app/views/jobseekers/profiles/employments/review.html.slim:8]: https://govuk-components.netlify.app/helpers/button/

Think you can just add an option:

`{ secondary: true }`

**Review by josephhull676 — APPROVED** [review]

File: app/views/jobseekers/profiles/employments/review.html.slim (hunk 1 above)
Comment (elliotcm) [reply app/views/jobseekers/profiles/employments/review.html.slim]: Done, thanks!

**Pull request merged**